surfer/viz.py

```diff
@@ -965,28 +965,26 @@ def add_data(self, array, min=None, max=None, thresh=None,
                  vertices=None, smoothing_steps=20, time=None,
                  time_label="time index=%d", colorbar=True,
                  hemi=None, remove_existing=False, time_label_size=14,
-                 initial_time=None):
+                 initial_time=None, scale_factor=None, vector_alpha=None):
         """Display data from a numpy array on the surface.
 
-        This provides a similar interface to add_overlay, but it displays
+        This provides a similar interface to
+        :meth:`surfer.viz.Brain.add_overlay`, but it displays
         it with a single colormap. It offers more flexibility over the
-        colormap, and provides a way to display four dimensional data
-        (i.e. a timecourse).
+        colormap, and provides a way to display four-dimensional data
+        (i.e., a timecourse) or five-dimensional data (i.e., a
+        vector-valued timecourse).
 
-        Note that min sets the low end of the colormap, and is separate
-        from thresh (this is a different convention from add_overlay)
-
-        Note: If the data is defined for a subset of vertices (specified
-        by the "vertices" parameter), a smoothing method is used to interpolate
-        the data onto the high resolution surface. If the data is defined for
-        subsampled version of the surface, smoothing_steps can be set to None,
-        in which case only as many smoothing steps are applied until the whole
-        surface is filled with non-zeros.
+        .. note:: ``min`` sets the low end of the colormap, and is separate
```

Comment:
This one seemed to be more likely to cause problems / be higher priority than the other one that was moved to `Notes`